en/identity-server/7.0.0/docs/apis/organization-apis/authentication.md

```diff
@@ -81,35 +111,52 @@ https://{{ host_name }}/oauth2/authorize?response_type=code&redirect_uri={redire
     <td><code>redirect_uri</code><Badge text="Required" type="mandatory"/></td>
     <td>This is where the response is redirected to at the end of the process. This needs to be the same as one of the URLs given in the registered apps.</td>
   </tr>
+  <tr>
+    <td><code>scope</code><Badge text="Required" type="mandatory"/></td>
+    <td>The scope corresponding to the API you want to use. See the relevant API reference docs for the list of internal scopes for each API.</td>
+  </tr>
   <tr>
     <td><code>client_id</code><Badge text="Required" type="mandatory"/></td>
     <td>The client ID obtained when registering the application in {{ product_name }}.</td>
   </tr>
 </table>
 
-## Get access tokens
-In this flow, the application needs to get tokens for the organization and exchange the obtained token to get an access token for the organization.
+## Step 5: Get access tokens
 
-Let's see how this works:
+After receiving the authorization code, the application has to exchange it to get the `id_token` and the `access_token`. The method of obtaining these tokens differs depending on the type of user which can be one of the following.
 
-### Step 1: For the organization (root)
-After receiving the authorization code, the application has to exchange it to get the tokens given below:
+  - A user that was created within the organization.
+  - A parent organization user that was invited to join the organization.
 
-- `access_token`
-- `id_token`
+### For a created user
```

Comment:
Better to reword this
+1 for https://github.com/wso2/docs-is/pull/4636#issuecomment-2202287279 suggestion 

Update in both doc spaces